model_compression_toolkit/core/common/quantization/quantization_params_generation/error_functions.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Callable
+import numpy as np
+import model_compression_toolkit.core.common.quantization.quantization_config as qc
+from model_compression_toolkit.core.common.quantization.quantization_params_generation.kl_selection import \
+    _kl_error_histogram, _kl_error_function
+from model_compression_toolkit.core.common.similarity_analyzer import compute_mse, compute_mae, compute_lp_norm
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+
+
+def _mse_error_histogram(q_bins: np.ndarray,
```

Comment:
maybe move all these functions (mse,mae, etc)  to a new file called histogram_similarity_analyzer.py 
And rename similarity_analyzer to tensor_similarity_analyzer.py